src/components/ListView/index.jsx

```diff
@@ -1,29 +1,23 @@
 import React from 'react';
 import { useCollection } from 'react-firebase-hooks/firestore';
 import { getProducts } from 'components/Utils/firestore';
+import EmptyList from 'components/EmptyList';
+import ProductList from 'components/ProductList';
 
 function ListView() {
   const query = getProducts();
   const [values, loading, error] = useCollection(query);
 
   return (
     <div>
-      <h2>List</h2>
-      <div>
-        {error && <strong>Error: {JSON.stringify(error)}</strong>}
-        {loading && <span>List: Loading...</span>}
-        {values && (
-          <span>
-            <div>
-              {values.docs.map((doc) => (
-                <div key={doc.id}>
-                  <p>{doc.data().item}</p>
-                </div>
-              ))}
-            </div>
-          </span>
-        )}
-      </div>
+      <h2>Smart Shopping List</h2>
+      {error && <strong>Error: {JSON.stringify(error)}</strong>}
+      {loading && <p>List: Loading...</p>}
+      {!loading && values.empty ? (
+        <EmptyList />
+      ) : (
+        <ProductList values={values} />
+      )}
```

Comment:
I like how you're using ternaries, nice job.

Reply:
Well! All your suggestions to change were changed!